lib/importer/src/index.js

```diff
@@ -477,6 +509,121 @@ exports.Initialise = (config, router, prototypeKit) => {
     },
   );
 
+  //--------------------------------------------------------------------
+  // Configuration management routes.
+  //--------------------------------------------------------------------
+
+  const TEMP_DIRECTORY_LABEL = "[Temporary directory]"
+
+  // Render the config page
+  router.get(
+    IMPORTER_ROUTE_MAP.get("importerConfiguration"),
+    (request, response) => {
```

Comment:
These config routes will still be functional even if the Prototype Kit is started in production mode, which normally removes access to routes under `/manage-prototype`. This would mean prototypes running on the web could be modified by their users. See https://github.com/alphagov/govuk-prototype-kit/blob/c0e707982e3e08dc5b43b7467196145fb96a29ba/lib/manage-prototype-routes.js#L50-L52